src/pydata_sphinx_theme/theme/pydata_sphinx_theme/layout.html

```diff
@@ -63,10 +63,10 @@
           id="pst-secondary-sidebar-checkbox"/>
   <label class="overlay overlay-secondary" for="pst-secondary-sidebar-checkbox"></label>
   {# A search field pop-up that will only show when the search button is clicked #}
-  <div class="search-button__wrapper">
-    <div class="search-button__overlay"></div>
-    <div class="search-button__search-container">{% include "../components/search-field.html" %}</div>
-  </div>
+  <dialog class="search-button__search-container">
```

Comment:
Wondering if this class should be renamed

Reply:
I have a question in another comments if this should even be made an id, as the js assume there is only one I think. Don't trust me on Js.